site/en/android-environment/lint-rules-for-ChromeOS-in-Android-Studio.md

```diff
@@ -0,0 +1,167 @@
+---
+title: ChromeOS lint rules in Android Studio
+metadesc: An in-depth guide on how to improve the experience of large screens on 
+ChromeOS with lint rules in Android Studio.
```

Comment:
```suggestion
metadesc: An in-depth guide on how to improve the experience of large screens on ChromeOS with lint rules in Android Studio.
```

Reply:
It is! I had updated that before but not synced, as there were still issues related to the images.